### ISSUE TITLE
Line 232: thoroughly confusing and undefined.

### ISSUE THREAD

**Issue by heroxbd** [heroxbd/waveform-analysis#111]

**Comment by TakaraResearch** [comment]
部分表达已经改变，进行概念介绍


3.4 Convolutional neural network

Neural Networks have made breakthroughs in various fields like computer vision and natural language processing. 
A commonly-seen neural network is an efficient computation structure composed of pointwise non-linear functions and weighted linear transformations with millions of parameters. It has a strong capability in expressing all varieties of functions and therefore outperforms many of the conventional algorithms. Based on a dataset made of collected data example pairs and a judge function called loss, parameters within a neural network could be updated via a method called back-propagation. Such a process is called training and a high precision mapping between feed-in data and their corresponding output is to be learned via parameter update through dataset processing. After a neural network is well trained, it could create a high accuracy output result corresponding to its feed-in input data. A strong prediction capability is established through such a mechanism and therefore the method finds its application in all walks of life.

Convolutional neural networks are a kind of neural network created with a structure of several layers. The calculation within each layer consists of a weighted addition process (link) and a point-wise nonlinear transformation (activation).  In the weighted addition process within a layer's calculation, only the neighboring outputs from the former layer are taken into account, and the parameters are shared between different positions to establish a local pattern recognition mechanism.
Such a 'neighbor-only shared' weighted addition is equivalent to a convolution calculation, and each set of parameters is called a kernel. Normally, multiple sets of parameters are used to express the multiple possible styles of combination

Only the output from the form The 'convolution' name co 
Each layer is maded
Within each layer, a weighted addition is 

Each layer is made of a convolutional weighted addition and a point-wise nonlinear transformation. The former is called lThe 'convolution' name is used because only neighbour 


consisted of weighted additions

established with millions of parameters via efficient compositions with weighted additions and pointwise non-linear function activations. It has a high  
It is established with efficient compositions of weighted additions and non-linear functions and has 
is a family of computation structures composed by 
outperforms many traditional algorithms






It As an efficient composition of weighted additions and non-linear functions, neural networks outperform many traditional algorithms.
We choose a shallow network structure of 5 layers to recognize patterns as shown in figure 7a, motivated by the pulse shape and universality of VPE(t) for all the PEs. The convolutional widths are selected considering the localized nature of VPE(t).
The workflow of data processing consists of training and predicting. We find a mapping from waveform w(t) to PE φ ̃(t) with the backpropagation method and supervised learning. We formulate the training loss as optimizing Dw[φˆ, φ ̃], the Wasserstein distance between the truth φ ̃ and predicted φˆ. As discussed in section 3.1.3, Dw can handle the PE sparsity in training iterations. Figure 7b shows the convergence of Wasserstein distance during training.

**Comment by heroxbd** [comment]
图有必要更新吗？

**Comment by TakaraResearch** [comment]
我觉得可能有必要添加一幅卷积神经网络的原理图
另外卷积神经网络的，使用的结构图 需要在描述上进行更新 在之后的修改中可能需要改的更标准一点

**Comment by heroxbd** [comment]
Takara_Research ***@***.***> writes:

> 我觉得可能有必要添加一幅卷积神经网络的原理图
> 另外卷积神经网络的，使用的结构图 需要在描述上进行更新 在之后的修改中可能需要改的更标准一点

可以和现有的图画在一起吗？

**Comment by TakaraResearch** [comment]
最好不要画在一起
卷积神经网络的原理图 侧重于原理
卷积神经网络的具体结构 可以表示这是一个例子 侧重网络具体结构 是现有图的细化
